### ISSUE TITLE
UnicodeWarning: Unicode unequal comparison failed to convert both arguments to Unicode - interpreting them as being unequal

### ISSUE THREAD

**Issue by karlp** [mailpile/Mailpile#1963]
After upgrading to latest git: Gunsmoke--TheLastApacheTag-1164-g9754cb3

```
/home/karlp/src/mailpile-git/mailpile/i18n.py:44: UnicodeWarning: Unicode unequal comparison failed to convert both arguments to Unicode - interpreting them as being unequal
  if t != string] + [string]
```

**Comment by JocelynDelalande** [comment]
Same here.

**Comment by karlp** [comment]
(it doesn't seem to _do_ anything, but presumably it's on the console for a reason)

**Comment by BjarniRunar** [comment]
Fixed in cb4cead4, thanks for reporting!
